assignments/assignment-8/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>RSVP</title>
+</head>
+<body>
+    <h1>RSVP Form</h1>
+    <h3>Join us for TuranTech's anniversary on April 1, 2024. Please reply to this RSVP by May 1st, 2023. Thank you, we hope to see you soon!</h3>
+
+    <form>
+        <label for="nameInput">Full Name</label>
+        <input type="text" name="fullName" id="nameInput">
```

Comment:
If asked fullName, the id attribute should be according to it like fullNameInput